backend/src/test/java/gov/cdc/usds/simplereport/api/BaseApiTest.java

```diff
@@ -42,36 +52,76 @@
     @Autowired
     private DbTruncator _truncator;
     @Autowired
-    private OrganizationInitializingService _initService;
+    protected OrganizationInitializingService _initService;
 
     @Autowired
     protected GraphQLTestTemplate _template; // screw delegation
 
+    @Autowired
+    protected AuthorizationProperties _authorizationProperties;
+    @Autowired
+    private OktaClientProperties _oktaClientProperties;
+
     @MockBean
     protected AuthorizationService _authService;
     @MockBean
     protected IdentitySupplier _supplier;
+    @MockBean
+    protected OktaService _oktaService;
 
-    @Autowired
-    private OktaService _oktaService;
+    protected Client _oktaClient;
```

Comment:
I'm a little confused about which things are mocked and which things are subject to setup/teardown, but that's probably a conversation to have after we've gotten past the current crisis (as long as we aren't doing too much weird stuff to our Okta directory in the process).

Reply:
yes, i admit i did the Absolute Minimum™️ to end the overly-invasive interaction with the Okta envt and to make the tests pass

Reply:
That seems like the right approach--I'm trying to take that approach as well, at least until we have our pinned tickets cleared. 🙂